lib/ace/layer/text.js

```diff
@@ -311,6 +319,21 @@ var Text = function(parentEl) {
             row++;
         }
         this.element.innerHTML = html.join("");
+        if (this.session.$bidiHandler.isCourierFontFace())
```

Comment:
If you would agree, I can just remove dynamic font change and let site owner to pick correct font.
But I would say that the font name suitable for Chrome should be somehow documented, isn't it?
May be in the the same editor.css